--- /home/gota/peachgk_md/peachgk-md-2.139/peachgk.ini	2015-01-06 12:40:39.604779000 +0900
+++ peachgk.ini	2015-01-06 14:24:53.188463600 +0900
@@ -1,4 +1,4 @@
-### MD control script for peachgk Ver.5.3 '14.06.20 ###
+### MD control script for peachgk Ver.5.4 '15.01.06 ###
 
 #input&output file name
 iuwtopname       H2O_topOB.dat
@@ -177,7 +177,9 @@
 mts_posres      short
 mts_potbias     long
 
-nstep_vir       10              ! number of step for virtual time integration
+iflimitmove     .false.         ! if limit atomic motion to a specific distance
+limitdist       0.1d-10         ! maximum atomic displacement when doing
+                                ! time integration [m] (structure relaxation)
 
 #Nose-Hoover chain and MTK eq. and higher order integration
 # for Nose-Hoover chain
@@ -237,10 +239,10 @@
 
 # parameter for SPME method
 #!!! FFT requires grid points are a multiple of 2,3,5
-nfft1           30              ! grid points in SPME
-nfft2           30              ! grid points in SPME
-nfft3           30              ! grid points in SPME
-pme_order       4               ! B-spline order
+nfft1           50              ! grid points in SPME
+nfft2           50              ! grid points in SPME
+nfft3           50              ! grid points in SPME
+pme_order       6               ! B-spline order
 
 # parameter for energy minimazation
 d_rini         0.5d-10          ! initial displacement dr for EM [m]
